src/pageObject.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
-
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return `Click on the ${this.header.logo}`;
+  }
 };
 
 class ArticlePage extends PageObject {
-
+  constructor(url, slug) {
+    super();
```

Comment:
Calling super() with no parameters in the ArticlePage class does not pass the url to the parent PageObject class. You should call super(url) instead.

Reply:
Done